app/konnect-platform/konnect-identity.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "{{site.konnect_short_name}} Identity"
+content_type: reference
+layout: reference
+tech_preview: true
+
+products:
+    - konnect-platform
+
+permalink: /konnect-identity/
+works_on:
+    - konnect
+search_aliases:
+  - Kong IdP
+  - Konnect IdP
+breadcrumbs:
+  - /konnect/
+
+api_specs:
+    - gateway/admin-ee
+    - konnect/control-planes-config
+
+related_resources:
+  - text: Configure the OIDC plugin with {{site.konnect_short_name}} Identity
+    url: /how-to/configure-konnect-identity-oidc/
+  - text: Configure the Upstream OAuth plugin with {{site.konnect_short_name}} Identity
+    url: /how-to/configure-konnect-identity-upstream-oauth/
+  - text: Configure the OAuth 2.0 Introspection plugin with {{site.konnect_short_name}} Identity
+    url: /how-to/configure-konnect-identity-oauth-introspection/
+
+description: |
+  {{site.konnect_short_name}} Identity enables you to use {{site.konnect_short_name}} to generate, authenticate and authorize API access. {{site.konnect_short_name}} Identity implements the OAuth2.0 standard with OpenID Connect for authentication and authorization. 
+
+faqs:
+  - q: How do I retrieve my client’s secret again?
+    a: |
+      To retrieve your client secret, you must access the credentials stored in the Service or plugin configuration. 
```

Comment:
I dont think credentials are stored in service or plugin either. @alexgervais can you confirm?